project-fortis-interfaces/src/components/Insights/MapBoundingReset.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import IconButton from 'material-ui/IconButton/IconButton';
+import Map from 'material-ui/svg-icons/maps/map';
+import { fullWhite } from 'material-ui/styles/colors';
+
+export default class MapBoundingReset extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      expanded: false
+    };
+  }
+
+  onClick = () => {
+    this.props.onClick();
+    this.setState({
+      expanded: !this.state.expanded
+    });
+  }
+
+  render() {
+    const { tooltipOn, tooltipOff, tooltipPosition } = this.props;
+    const { expanded } = this.state;

```

Comment:
Given that the code-paths for expanded and not-expanded are the same (this is just an action button, not a state button), why not remove this and the tooltipOn/tooltipOff distinction?

Reply:
you're totally right, just an oversight on my part. will fix shortly.